examples/quickstarts/helidon-quickstart-mp/pom.xml

```diff
@@ -50,6 +50,11 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.1</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.jboss.jandex</groupId>
+                    <artifactId>jandex-maven-plugin</artifactId>
+                    <version>1.0.5</version>
```

Comment:
If you want to (or need to) use Jandex 2.1.1 (@tjquinno?) then this must be version 1.0.6.

Reply:
Good point. We moved to 2.1.1 in the workspace. I'll update these.

Reply:
Our build (top-level project pom.xml) specifies Jandex 2.1.1.Final. Laird, do you have a reference for the 1.0.6 requirement for 2.1.1.Final?

Reply:
@ljnelson ^

Reply:
Yep; see https://github.com/wildfly/jandex-maven-plugin/commits/master and read up from April 3, 2019.

Reply:
Because we specified 2.1.1.FInal already in our top-level pom's dep. mgt. section does that influence the plug-ins' dependencies too?

Reply:
No.  If you wanted to effectively use 1.0.5 for some reason with Jandex 2.1.1, thus effectively creating a very clean Frankenstein jandex-maven-plugin that is exactly equivalent to 1.0.6 as far as I can tell, you would add a `<dependencies>` stanza to the relevant `<plugin>` element in `<pluginManagement>` that referenced Jandex 2.1.1.  I think it's simpler just to change the jandex-maven-plugin version.

Reply:
Oh, yes. I wasn't suggesting we rely on that (even if it did work). I was just curious.

Reply:
I've updated the jandex plugin version in the top level pom.xml, and the jandex versions in the quickstart MP pom.xml. I've also updated the bookstore test app to use the jandex plugin.